CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Version 1.3.3
+
+*June 24, 2021*
+
+There was an inconsistency for Diff-Orders and for Orders API. The "t" field was being described as number 0 for selling
+and 1 for buying, but the implementation of this api is validating the opposite. 
+To make the documentation consistent with the code, the field was updated to number 0 for buying and number 1 for selling.
+More details can be found in this task on Jira: https://bitsomx.atlassian.net/browse/AS2-51
```

Comment:
As this documentation is going to be public available I wouldn't recommend adding the JIRA link

Reply:
Hey @viccomx, like Alex said, there are two changelogs: an internal and an external. This _CHANGELOG.md_ file is internal only, so this will not be shown in api_info page. 
The external changelog is this file: _source/index.html.md_

Reply:
Hi @matheusdealencar , I saw the history of the file and Jira links are not included, so, just to keep consistency could you please remove this line?

Reply:
> Hi @matheusdealencar , I saw the history of the file and Jira links are not included, so, just to keep consistency could you please remove this line?

Done. Please review it again. Also, @viccomx and @thenewversion